.gitignore

```diff
@@ -16,6 +16,7 @@ env.d/*
 # Tests temporary files
 .coverage
 .pytest_cache
+.pylint.d
```

Comment:
You should complete your description explaining why you use `-d` and what is `-d` option.

Reply:
It should be mentioned in the documentation (README) or do you have something else in mind?

Reply:
I mean commit description

Reply:
I propose the following:

```
(project) use arnold -d run to execute dev commands

Instead of having x scripts to run development commands, use a unique
entrypoint: bin/arnold using the -d option to activate the `DEV_MODE`

An active `DEV_MODE` implies the following behavior:

- `docker run` commands mount the local directory sources as a volume in `/app`, or,
- `docker run` commands mount the local application tree in `/app/apps` if a tray as been found in the repository

Makefile rules have been added to run tests and linters.
```

WDYT?

Reply:
ok for me